### ISSUE TITLE
`RuntimeException` with pixies.fs, pixie.io

### ISSUE THREAD

**Issue by ruivieira** [pixie-lang/pixie#517]
OS: OS X 10.11.5
pixie-vm: 0.1 built from master (dd754fe9f329e9e176eeaedae1095c85cde65028)

With this simple script:
```
(ns hello
  (:require [pixie.fs :refer :all]
    [pixie.io :as io])
  )

(defn show-files [d]
  (let [files (io/walk-files (dir d))]
     (for [file files] (println file))
  )
)
```

When running `(show-files ".")`, works as expected.

When running `(show-files "../clojure")` VM throws:

```
RuntimeException: :pixie.stdlib/InternalError Internal error: <UnicodeDecodeError object at 0x10057d488>

[1]    30967 segmentation fault  pixie-vm hello.pxi
```

**Comment by alekcz** [comment]
@ruivieira I ran your script in both the `pixie` repl and the `dust` repl. I couldn't reproduce your error. I'm running Pixie v0.1 on OS X 10.11.4.

That sounds like an character encoding/decoding error. 

What does running `$ locale` in the terminal return? 
Also is what encoding is `hello.pxi` encoded in?

I'm gonna build [dd754fe](https://github.com/pixie-lang/pixie/commit/dd754fe9f329e9e176eeaedae1095c85cde65028). I'll let you know how that goes. I doubt it's pixie version error though.

**Comment by ruivieira** [comment]
@alekcz, thanks.

Running `locale`:
```
LANG="en_GB.UTF-8"
LC_COLLATE="en_GB.UTF-8"
LC_CTYPE="en_GB.UTF-8"
LC_MESSAGES="en_GB.UTF-8"
LC_MONETARY="en_GB.UTF-8"
LC_NUMERIC="en_GB.UTF-8"
LC_TIME="en_GB.UTF-8"
LC_ALL=
```
And `file -I hello.pxi`:
```
hello.pxi: text/plain; charset=us-ascii
```

**Comment by alekcz** [comment]
@ruivieira
TL;DR There is a file or directory that has a non-ASCII character in its name.

I rebuilt pixie. It's not a version issue.  I did reproduce the error though. 
```clojure
touch ñotascii
pixie-vm hello.pxi
```
Output: 
```
RuntimeException: :pixie.stdlib/InternalError Internal error: 
<UnicodeDecodeError object at 0x100580450>
```
The error comes from trying to print a non-ASCII. e.g. `(println "ñotascii")`
There is a file or folder that has a character that is not part of the ASCII character set in its name. 
In your case it should be somewhere in `../clojure`.

**Comment by ruivieira** [comment]
That was fast!

Thanks.
